### PR TITLE
Feat/added env vars runtime config

### PR DESCRIPTION
**Pull request by makarofalex3**
Added env vars runtime config in FE part, need update BE and CI/CD part

from this step:  https://github.com/kunokdev/cra-runtime-environment-variables#-production

### CODE REVIEW

**Comment by Dzeranov** [comment]
Please, update `README`.
And why do we need to update BE part?

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "yup": "^0.32.9"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "chmod +x ./env.sh && ./env.sh && cp env-config.js ./public/ && react-scripts start",
     "start-prod": "node server.js",
```
Comment (Dzeranov) [inline package.json:47]: We need to change `start-prod` command too.

**Comment by makarofalex3** [comment]
@ivhus

**Review by ivhus — CHANGES_REQUESTED** [review]
Chechking on the topic found this documentation https://nodejs.dev/learn/how-to-read-environment-variables-from-nodejs 

This approach by the link is what's needed

**Pull request closed without merging**